plugins/woocommerce/includes/data-stores/class-wc-product-data-store-cpt.php

```diff
@@ -642,21 +642,21 @@ protected function update_post_meta( &$product, $force = false ) {
 					// Fire actions to let 3rd parties know the stock is about to be changed.
 					if ( $product->is_type( 'variation' ) ) {
 						/**
-						* Action to signal that the value of 'stock_quantity' for a variation is about to change.
-						*
-						* @since 4.9
-						*
-						* @param int $product The variation whose stock is about to change.
-						*/
+						 * Action to signal that the value of 'stock_quantity' for a variation is about to change.
+						 *
+						 * @param WC_Product $product The variation whose stock is about to change.
+						 *
+						 * @since 4.9
+						 */
 						do_action( 'woocommerce_variation_before_set_stock', $product );
 					} else {
 						/**
-						* Action to signal that the value of 'stock_quantity' for a product is about to change.
-						*
-						* @since 4.9
-						*
-						* @param int $product The product whose stock is about to change.
-						*/
+						 * Action to signal that the value of 'stock_quantity' for a product is about to change.
+						 *
+						 * @param WC_Product $product The product whose stock is about to change.
```

Comment:
Curious why flip this one?, as I can see the pattern is @since is above @param in other places.

Reply:
I think I must have copied the block from somewhere else before updating the values. It's actually pretty common to have `@since` be the last thing in the doc block, if you do a regex search in the codebase with something like `\* @since .*\n\t+ \*/` there are lots of examples.

It would be nice if we had a standard for the order that doc block attributes should be in, but I don't think that exists at the moment...